### ISSUE TITLE
Module not found "./index.js" on bun run start

### ISSUE THREAD

**Issue by frogconn** [gornostay25/svelte-adapter-bun#2]
After build with vite and start app by ```bun run start```
showing error error: Module not found "./index.js" in build dir.

![image](https://user-images.githubusercontent.com/2070583/178396826-025239b9-9210-465f-a8fd-37a97e7a403d.png)

When i try to run

```bun build/server/index.js```

or

```bun run start build/server/index.js```

show errror here.

![image](https://user-images.githubusercontent.com/2070583/178397628-eaf9e5fc-ddcc-4f96-9f21-e8f404fa106d.png)

**This app create by default todo app from https://kit.svelte.dev/**

**Comment by gornostay25** [comment]
Yes, it's my mistake. i was published a packet of without files directory... I fixed it, update the packet version to 0.2.2-3
